packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721Factory.sol

```diff
@@ -14,8 +14,13 @@ contract OptimismMintableERC721Factory is Semver {
      *
      * @param localToken  Address of the token on the this domain.
      * @param remoteToken Address of the token on the remote domain.
+     * @param deployer    Address of the initiator of the deployment
      */
-    event OptimismMintableERC721Created(address indexed localToken, address indexed remoteToken);
+    event OptimismMintableERC721Created(
+        address indexed localToken,
+        address indexed remoteToken,
+        address deployer
```

Comment:
Comment only: It feels like this should've been indexed in the ERC20 factory, but better to keep consistent for now.